create-assigned-issue.rb

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env ruby
+banner = <<-BANNER
+Usage: create-assigned-issue.rb <issue URL>
+  * Copies issue as template
+  * Replaces any references to the assigned user (eg, at-mentions, etc.)
+  * Also "undoes" any checkboxes in issue template (eg, [x] -> [ ])
+  * For each collaborator of repo, creates a new issue, assigned to collaborator
+BANNER
+
+$stderr.sync = true
+require 'octokit'
+require 'optparse'
```

Comment:
hahaha consistency.

Yeah, that makes sense and it does read real well for that. Okay good motivation. Thanks!